Apache-Beam-Jobs.md

```diff
@@ -227,6 +229,257 @@ error_pcoll = (
 
 `Runner`s provide the `run()` method used to visit every node (`PValue`) in the pipeline's DAG by executing the edges (`PTransform`s) to compute their values.  At Oppia, we use `DataflowRunner` to have our `Pipeline`s run on the [Google Cloud Dataflow service](https://cloud.google.com/dataflow).
 
+## Writing Apache Beam Jobs
+
+For this section, we'll walk through the steps of implementing a job by writing one: `CountExplorationStatesJob`.
+
+Here's the DAG we'll try to implement:
+
+    .--------------. Count states .--------. Sum .-------.
+    | Explorations | -----------> | Counts | --> | Total |
+    '--------------'              '--------'     '-------'
+
+### 1. Subclass the `base_jobs.JobBase` class and override the `run()` method
+
+Make sure your job class name is clear and concise, because the name is presented to release coordinators:
+
+<!-- TODO(#11475): Replace this screenshot with one that has job names which follow our naming convention. -->
```

Comment:
I don't see this tracked in #11475.

It's also on a separate repo so maybe better to just fix it (through in-browser html manipulation) and get rid of the todo?

Reply:
Done.